.github/workflows/issue-tests.yml

```diff
@@ -20,13 +20,27 @@ jobs:
 
     runs-on: ubuntu-22.04
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
 
     - name: Set up Go
-      uses: actions/setup-go@v3
+      uses: actions/setup-go@v4
       with:
         go-version: "1.23.1"
 
+    - name: Cache Go modules
```

Comment:
curious - did this bring about a significant difference? my guess is it did not?